tests/xcconfig/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+load("//rules:framework.bzl", "apple_framework")
+load(":tests.bzl", "xcconfig_unit_test_suite")
+
+SRCS = [
+    "empty.m",
+    "empty.swift",
+]
+
+EXPECTED_FAILURE = ["manual"]
+
+apple_framework(
+    name = "empty_xcconfig",
+    srcs = SRCS,
+    xcconfig = {},
+)
+
+apple_framework(
+    name = "unknown_key_xcconfig",
+    srcs = SRCS,
+    xcconfig = {
+        "MY_MADEUP_SETTING": "NEVER",
+    },
+)
+
+apple_framework(
+    name = "boolean_key_xcconfig",
+    srcs = SRCS,
+    xcconfig = {
+        "GCC_TREAT_WARNINGS_AS_ERRORS": "YES",
+    },
+)
+
+# apple_framework(
+#     name = "invalid_boolean_key_xcconfig",
+#     srcs = SRCS,
+#     xcconfig = {
+#         "GCC_TREAT_WARNINGS_AS_ERRORS": "FILE_NOT_FOUND"
+#     },
+# )
+
+genquery(
+    name = "boolean_key_xcconfig_libraries",
```

Comment:
How is the output file generated by this genquery being used?

Reply:
It’s not, I can remove it, but it’s a way of seeing what copts are being passed through to the underlying rules